### ISSUE TITLE
loss value

### ISSUE THREAD

**Issue by yjcaimeow** [jchibane/if-net#38]
Dear author,

I retrained the shapenet model with spare point cloud as input. The initial loss values are about 20000. Is it normal since I think the loss is really large.

Do you remember the final loss value?

Best regards,
Yingjie CAI

**Comment by Bingoang** [comment]
@yjcaimeow  Hi, I'm  also concerned about this issue. And what are your loss values in the later stage epochs?

**Comment by XuHQ1997** [comment]
@Bingoang @jchibane I'm training Voxel Super-Resolution task based on this code, and also concered about this issue. It seems the loss values summed over all points of a shape, so the value is strangely large.

Have you trained this network? And what are your loss values after a long time training?

Looking forward for your reply. Thanks.

**Comment by Bingoang** [comment]
@XuHQ1997 
Hi, in my training(within 100 epochs): For Voxel32, the minimum validation loss is about 12000; For Voxel128, the minimum validation loss is about 3200. The loss value looks very large, but the reconstruction looks reasonable.

**Comment by YangZiz** [comment]
Hello, I recently reproduced this paper. When I was training on my own training set, I ran a few epochs and had an overfitting problem. Have you ever had this situation？
